### PR TITLE
[spine-runtimes] create a new port

### PR DESCRIPTION
**Pull request by luncliff**
### What does your PR fix?

Build C/C++ library part of https://github.com/EsotericSoftware/spine-runtimes/

It uses the [Spine Runtimes license](http://ko.esotericsoftware.com/spine-runtimes-license), which is non-SPDX. I left `"license"` field empty.

### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?

Targets Windows, UWP triplets.

* `x64-windows`
* `arm64-windows`
* `x64-uwp`

### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

I hope so.

### CODE REVIEW

**Comment by autoantwort** [comment]
> It uses the [Spine Runtimes license](http://ko.esotericsoftware.com/spine-runtimes-license), which is non-SPDX. I left "license" field empty.

Then you should set it to `null`

File: ports/spine-runtimes/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,34 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY) # the project doesn't support SHARED
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO EsotericSoftware/spine-runtimes
+    REF 4.1.00
+    SHA512 40a352a1f5e9939802667f330c9de2f0b03bf63834d1c20514a6cecb35c1a9915fb13588ee92cbaba9effbd2205c25851cca58d2ec7f90ce9b974252bd168425
+    PATCHES
+        fix-cmake.patch
+    HEAD_REF 4.1
```
Comment (LilyWangLL) [inline ports/spine-runtimes/portfile.cmake:10]:
```suggestion
    HEAD_REF 4.1
    PATCHES
        fix-cmake.patch
```

Comment (LilyWangLL) [reply ports/spine-runtimes/portfile.cmake:10]: Format suggestion, new patches may be added in the future.

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

**Review by LilyWangLL — DISMISSED** [review]

**Comment by vicroms** [comment]
I took a look at `spine-runtimes`' CMakeLists.txt and enabling `SPINE_SFML` will make `spine-runtimes` [vendor their own copy of SFML](https://github.com/EsotericSoftware/spine-runtimes/blob/4.1/spine-sfml/c/CMakeLists.txt#). 

We should either modify their CMakeLists.txt to accept vcpkg's installed `sfml` or remove the feature from the port and disable it explicitly.

**Comment by luncliff** [comment]
Then I will delete the feature.
Thanks for the check!

**Pull request merged**